source/index.txt

```diff
@@ -11,32 +11,13 @@ The MongoDB |version| Manual
 
    For new features in MongoDB 4.2 see :doc:`/release-notes/4.2`.
 
-Welcome to the MongoDB |version| Manual! MongoDB is a
-document database designed for ease of development
-and scaling. The Manual introduces key concepts in MongoDB, presents the
-query language, and provides operational and administrative
-considerations and procedures as well as a comprehensive reference
-section. The manual is also available as :hardlink:`HTML tar.gz
-<manual.tar.gz>` and :hardlink:`EPUB <MongoDB-manual.epub>`.
-
-MongoDB offers both a *Community* and an *Enterprise* version of the
-database:
-
-- MongoDB Community is the {+source-available+} edition of MongoDB.
-
-- MongoDB Enterprise is available as part of the MongoDB Enterprise
-  Advanced subscription and includes comprehensive support for your
-  MongoDB deployment. MongoDB Enterprise also adds enterprise-focused
-  features such as LDAP and Kerberos support, on-disk encryption,
-  and auditing.
-
-MongoDB also offers
-`Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_server>`__,
-a hosted MongoDB Enterprise service option in the cloud which requires
-no installation overhead and offers a free tier to get started.
-
-The Manual documents both MongoDB Community and Enterprise features and
-functionality.
+   To download the MongoDB 4.2, go to the `MongoDB Download Center
```

Comment:
I noticed this was missed on only the 4.2 branch so I added the download blurb back in this branch.

Reply:
I also noticed that 4.2 and 4.0 have the links https://www.mongodb.com/download-center?jmp=docs#production where as 4.4,5.0,5.1 have https://www.mongodb.com/try/download?tck=docs_server would this be worth another ticket for update?